lol.js

```diff
@@ -0,0 +1 @@
+console.log("YOU HAVE TO REMOVE THIS SORRY!")
```

Comment:
@KlevisImeri Blah blah code code.

Try doing this instead:

```suggestion
console.log("YOU HAVE TO REMOVE THIS NOT SORRY!")
```

Also pls include #2 and #1 or something.

🥇

Reply:
Fixed #2